content/styles/01-atoms/buttons.css

```diff
@@ -25,17 +25,38 @@ input[type=submit] {
 button:hover,
 input[type=submit]:hover {
   color: var(--white);
-  background-color: var(--primary-color);
+  background-color: var(--secondary-color);
+  border-color: var(--secondary-color);
 }
 
-.reverse {
-  color: var(--white);
-  background-color: var(--primary-color);
-  border: .125rem solid var(--primary-color);
+.read {
+  display: flex;
+  justify-content: center;
+  padding: .5rem 2.5rem;
+  max-width: 50%;
+  margin: 2rem auto 0;
 }
 
-.reverse:hover {
-  color: var(--primary-color);
-  background-color: var(--white);
-  border: .125rem solid var(--primary-color);
+.read::before {
+  margin-right: 2rem;
+  font-family: "fontawesome solid";
+  font-size: 2rem;
+  content: "\f518";
+}
+
+.sign {
+  background-color: var(--primary-sign-color);
```

Comment:
Is primary-sign-color going to be used to something else than sign?
If yes, the name might be irrelevant,
If not, the variable might be useless and the colors should be hardcoded here

Reply:
this color (just a test for now) could be use for action with strong implication like publish, become a neomad, sign in, ... I know the name is pretty irrelevant but I don't have any idea.

It's an action, but I want something more strong to catch user eye et put the other regular link more quiet and smooth/soft.

Any idea on your side ??

Reply:
Suggestions for brainstorming:
- `primary-highlight-color`
- `primary-action-color`
- `primary-strong-color`
- …